andela/ah-commando-frontend
# #168385309 category articles

#### What does this PR do?
It enables a user to get articles in a particular category
#### Description of Task to be completed?
- created a category page component
#### How should this be manually tested?
-  visit the staging link
- Click on more button of any of the categories to see more articles
#### Any background context you want to provide?
[E2e Link](https://drive.google.com/file/d/1KKVvFu_cGRCN-yXuqumxLEBYrDJq1oQ5/view?usp=sharing)
#### What are the relevant pivotal tracker stories?
[#168385309](https://www.pivotaltracker.com/story/show/168385309)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
